flink-runtime/src/main/scala/org/apache/flink/runtime/jobmanager/JobManager.scala

```diff
@@ -149,6 +154,9 @@ class JobManager(
 
   var leaderSessionID: Option[UUID] = None
 
+  private var metricsRegistry : FlinkMetricRegistry = _
+  private var jobManagerMetricGroup : JobManagerMetricGroup = _
```

Comment:
We could maybe create the `JobManagerMetricGroup` directly in the constructor of the `JobManager` if we pass in the `FlinkMetricRegistry` as a constructor argument.